src/stdlib/Localization.ts

```diff
@@ -51,22 +52,24 @@ function parseTranslations(xmlNode: XmlDocument, locale: string) {
  * @param rootDir The root package directory
  */
 export async function loadTranslationFiles(interpreter: Interpreter, rootDir: string) {
-    locales.forEach(async (locale) => {
-        const filePath = path.join(rootDir, "locale", locale, "translations.ts");
-        if (fs.existsSync(filePath)) {
-            let xmlNode: XmlDocument;
-            try {
-                let contents = await readFile(filePath, "utf-8");
-                let xmlStr = contents.toString().replace(/\r?\n|\r/g, "");
-                xmlNode = new XmlDocument(xmlStr);
-            } catch (err) {
-                interpreter.stderr.write(`Error reading translations file ${filePath}: ${err}`);
-                return;
-            }
+    await pSettle(
+        Array.from(locales).map(async (locale) => {
+            const filePath = path.join(rootDir, "locale", locale, "translations.ts");
+            if (fs.existsSync(filePath)) {
+                let xmlNode: XmlDocument;
+                try {
+                    let contents = await readFile(filePath, "utf-8");
+                    let xmlStr = contents.toString().replace(/\r?\n|\r/g, "");
+                    xmlNode = new XmlDocument(xmlStr);
+                } catch (err) {
+                    interpreter.stderr.write(`Error reading translations file ${filePath}: ${err}`);
+                    return;
+                }
 
-            parseTranslations(xmlNode, locale);
-        }
-    });
+                parseTranslations(xmlNode, locale);
```

Comment:
We could probably move this `parseTranslations()` call into the `try` branch since `catch` returns early anyway?